### PR TITLE
19807 More config fixes

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#19807

*Description of changes:*
More config fixes/additions

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-filer/devops/vaults.gcp.env [hunk 1]
```diff
@@ -31,4 +31,8 @@ NATS_CLIENT_NAME="op://nats/$APP_ENV/nats-filer/NATS_CLIENT_NAME"
 NATS_FILER_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_FILER_SUBJECT"
 NATS_ENTITY_EVENT_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_ENTITY_EVENT_SUBJECT"
 NATS_EMAILER_SUBJECT="op://nats/$APP_ENV/nats-emailer/NATS_EMAILER_SUBJECT"
+AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
```
Comment (JazzarKarim) [inline queue_services/entity-filer/devops/vaults.gcp.env:34]: I'm wondering about this /payment directory in 1Pass. Maybe we need one for each queue service? Especially for the TOPIC_NAME below.

File: jobs/future-effective-filings/devops/vaults.gcp.env [hunk 2]
```diff
@@ -1,7 +1,7 @@
 BUSINESS_API_URL="op://API/$APP_ENV/business-api/BUSINESS_API_URL"
 BUSINESS_API_VERSION2="op://API/$APP_ENV/business-api/BUSINESS_API_VERSION_2"
-AUDIENCE=op://gcp-queue/$APP_ENV/payment/AUDIENCE"
-TOPIC_NAME=op://gcp-queue/$APP_ENV/payment/TOPIC_NAME"
-PUBLISHER_AUDIENCE=op://gcp-queue/$APP_ENV/payment/PUBLISHER_AUDIENCE"
-GCP_AUTH_KEY=op://gcp-queue/$APP_ENV/payment/GCP_AUTH_KEY"
+AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
+TOPIC_NAME="op://gcp-queue/$APP_ENV/payment/TOPIC_NAME"
+PUBLISHER_AUDIENCE="op://gcp-queue/$APP_ENV/payment/PUBLISHER_AUDIENCE"
+GCP_AUTH_KEY="op://gcp-queue/$APP_ENV/payment/GCP_AUTH_KEY"
```
Comment (argush3) [inline jobs/future-effective-filings/devops/vaults.gcp.env:6]: everything is linked to the `payment` vault.  is this right?

Comment (JazzarKarim) [reply jobs/future-effective-filings/devops/vaults.gcp.env:6]: Was just wondering about that. My first guess would be yes. The Audience is always "https://pubsub.googleapis.com/google.pubsub.v1.Subscriber" and the PUBLISHER_AUDIENCE is always "https://pubsub.googleapis.com/google.pubsub.v1.Publisher". The same is for GCP AUTH KEY. All is good.

Now, the TOPIC_NAME, I think should be different.

File: queue_services/entity-filer/devops/vaults.gcp.env (hunk 1 above)
Comment (argush3) [reply queue_services/entity-filer/devops/vaults.gcp.env:34]: yeah, we should probably add sections in 1pass for each queue i think.

File: jobs/future-effective-filings/devops/vaults.gcp.env (hunk 2 above)
Comment (argush3) [reply jobs/future-effective-filings/devops/vaults.gcp.env:6]: there is this, so we could use that 
image

Comment (JazzarKarim) [reply jobs/future-effective-filings/devops/vaults.gcp.env:6]: Yup, exactly. Thanks!

**Review by argush3 — APPROVED** [review]

**Pull request merged**